rand_pcg/tests/lcg128xsl64.rs

```diff
@@ -1,6 +1,18 @@
 use rand_core::{RngCore, SeedableRng};
 use rand_pcg::{Lcg128Xsl64, Pcg64};
 
+#[test]
+fn test_lcg128xsl64_advancing() {
+    let seed = Default::default();
+    let mut rng1 = Lcg128Xsl64::from_seed(seed);
+    let mut rng2 = Lcg128Xsl64::from_seed(seed);
+    for _ in 0..20 {
+        rng1.next_u64();
+    }
+    rng2.advance(20);
+    assert_eq!(rng1.next_u64(), rng2.next_u64());
+}
```

Comment:
1. Is it sufficient to check only a single seed?
2. Might be clearer to write `let rng2 = rng1.clone()`
3. The generators support `Eq` so we can (more completely) test them with `assert_eq!(rng1, rng2)`

Reply:
I guess we could use `seed_from_u64` for 10 different integers.